galasa-maven-plugin/README.md

```diff
@@ -0,0 +1,198 @@
+# Maven Plugin for building Galasa
+
+This repository is an extension to Maven2 and includes goals that are used to build OSGI bundles and test catalogs for Galasa. The OSGI bundle repositories contain all the project information, configuration details and dependencies that are needed for building and running Galasa projects. The test catalog is used to manage Galasa test cases - attributes associated with test cases that are held in the catalog can used to schedule test runs.
+
+## How to build locally
+Use the `build-locally.sh` script to build this code locally.
+
+Environment variable over-rides:
+
+- `LOGS_DIR` - Optional. Where logs are placed. Defaults to creating a temporary directory.
+- `SOURCE_MAVEN` - Optional. Where a maven repository is from which the build will draw artifacts.
+- `DEBUG` - Optional. Defaults to 0 (off)
+- `GPG_PASSPHRASE` - Used to sign and verify artifacts during the build
+
+## How to use the plugin
+
+### Building a test catalog for a Java bundle
```

Comment:
These headings haven't rendered for some reason

Reply:
Fixed